README.md

```diff
@@ -150,6 +150,52 @@ const [gql, resolver] = usecase2mutation(usecase, resolverFunc)
 ​
 Or you can use `herbs2gql` [`defaultResolver`](https://github.com/herbsjs/herbs2gql/blob/master/src/defaultResolver.js) implementation as a reference. 
 ​
+#### Error Handling
+
+`herbs2gql` deals with errors in the default resolver. It translates the usecase's errors into graphql errors. By default, it throws `UserInputError` for `invalid arguments` or `invalid entities` errors, `ApolloError` for `not found`, `already exists`, and `unknown` errors, and `ForbiddenError` for `authorization` errors and any other kind of errors.
```

Comment:
here a table is better to represent correlation, what do you think about?

Reply:
Done!